### PR TITLE
terragrunt: add page

### PR DESCRIPTION
**Pull request by gcagle3**
- [X] The page (if new), does not already exist in the repository.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

*Version of the command being documented (if known):* 0.32.0+

### CODE REVIEW

**Comment by navarroaxel** [comment]
Hi @gcagle3, could you please sign the license to pass the CLA check?

**Comment by gcagle3** [comment]
> Hi @gcagle3, could you please sign the license to pass the CLA check?
> 
> Maybe you need to amend your commit using:
> 
> ```shell
> git commit --amend --author 'Your name <user@example.com>'
> git push --force
> ```
> 
> And the name and email must match your GitHub account.

Should be set now!

File: pages/common/terragrunt.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# terragrunt
+
+> Keep your Terraform CLI arguments DRY.
+> More information: <https://terragrunt.gruntwork.io/>.
```
Comment (navarroaxel) [inline pages/common/terragrunt.md:4]:
```suggestion
> More information: <https://terragrunt.gruntwork.io>.
```

File: pages/common/terragrunt.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Show module output values:
+
+`terragrunt output`
+
+- Destroy Terraform-managed infrastructure:
+
+`terragrunt destroy`
+
+- Build or change infrastructure from a tree of terragrunt modules (stack):
```
Comment (navarroaxel) [inline pages/common/terragrunt.md:26]: what do stack means?
```suggestion
- Build or change infrastructure from a tree of Terragrunt modules (stack):
```

Comment (gcagle3) [reply pages/common/terragrunt.md]: 'Stack' is used to refer to a tree of Terragrunt modules.

**Review by CleanMachine1 — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**